frontend/src/lib/components/Errors/ErrorDisplay.tsx

```diff
@@ -162,18 +205,20 @@ export function ErrorDisplay({ eventProperties }: { eventProperties: EventType['
                 />
                 <TitledSnack title="synthetic" value={$exception_synthetic ? 'true' : 'false'} />
                 <TitledSnack title="library" value={`${$lib} ${$lib_version}`} />
-                <TitledSnack title="browser" value={`${$browser} ${$browser_version}`} />
-                <TitledSnack title="os" value={`${$os} ${$os_version}`} />
+                <TitledSnack title="browser" value={$browser ? `${$browser} ${$browser_version}` : 'unknown'} />
+                <TitledSnack title="os" value={$os ? `${$os} ${$os_version}` : 'unknown'} />
             </div>
-            {!!$exception_stack_trace_raw?.length && (
+            {$exception_list?.length ? (
+                <ChainedStackTraces exceptionList={$exception_list} />
```

Comment:
sentry reverses of the `exception_list` in the server (historical reasons).
https://develop.sentry.dev/sdk/event-payloads/exception/

> Multiple values represent chained exceptions and should be sorted oldest to newest. For example, consider this Python code snippet:

I think the SDK has to capture already in the right order, and the parsing/showing has to maintain that order, I am not sure this is correct right now assuming that we don't do the reverse either.

Reply:
In python we effectively reverse so the first exception that happens is at the top. Maintaining this invariant in the SDKs this should be ok - as long as  every SDK of ours does the same?

Reply:
I think each platform has to decide whether should reverse or not (this varies depending on the platform).
So if there's a list of [exp 1, exp 2] in the JSON, it's shown as exp 1 on the top and exp 2 on the bottom.
If Python puts the exp 1 on the top or the bottom, then it depends on how Python itself displays chained exceptions in the console, I'd just follow their format, so the SDK decides the sorting, and the backend|frontend maintains it.